typescript/infra/config/environments/mainnet3/warp/configGetters/getArtelaBaseSolanaARTWarpConfig.ts

```diff
@@ -0,0 +1,47 @@
+import { ethers } from 'ethers';
+
+import {
+  ChainMap,
+  HypTokenRouterConfig,
+  OwnableConfig,
+  TokenType,
+} from '@hyperlane-xyz/sdk';
+
+import { RouterConfigWithoutOwner } from '../../../../../src/config/warp.js';
+import { SEALEVEL_WARP_ROUTE_HANDLER_GAS_AMOUNT } from '../consts.js';
+
+const ARTELA_OWNER = '0x801e8135867D65e742eb070A9fC0aD9c2f69B4cd';
+const ART_ON_SOLANA_ADDRESS = 'ELAJhVNCRfipNT99YTfPBGTAgyD5x9mEv3DYr9fvRM2C';
+const ISM_CONFIG = ethers.constants.AddressZero; // Default ISM
+export const getArtelaBaseSolanaARTWarpConfig = async (
+  routerConfig: ChainMap<RouterConfigWithoutOwner>,
+  abacusWorksEnvOwnerConfig: ChainMap<OwnableConfig>,
+): Promise<ChainMap<HypTokenRouterConfig>> => {
+  const artela: HypTokenRouterConfig = {
+    ...routerConfig.artela,
```

Comment:
nit: I think nam is trying to move us over to doing this instead now:
```
mailbox: routerConfig.artela.mailbox,
```

instead of the whole routerConfig